src/distilabel/llms/huggingface/inference_endpoints.py

```diff
@@ -316,11 +320,28 @@ async def agenerate(  # type: ignore
                 `0.0` nor `1.0` are valid values in TGI.
             top_p: the top-p value to use for the generation. Defaults to `1.0`.
             typical_p: the typical-p value to use for the generation. Defaults to `0.5`.
+            stop_sequences: either a single string or a list of strings containing the sequences
+                to stop the generation at. Defaults to `None`, but will be set to the
+                `tokenizer.eos_token` if available.
 
         Returns:
             A list of lists of strings containing the generated responses for each input.
         """
 
+        if stop_sequences is None and self._tokenizer is not None:
+            stop_sequences = [self._tokenizer.eos_token]
```

Comment:
Do we need this? In theory, default `stop_sequences` should be loaded automatically, right?